features/uri-templating.feature

```diff
@@ -13,95 +13,61 @@ Functionaliteit: URI templating
-    In scenario 'url templating is toegepast in de server url' wordt voor de Server objecten URI templating toegepast.
-
     Afspraken:
     - gebruik {serverurl} als placeholder voor de server url van externe API url's
     - gebruik {propertynaam} als placeholder in een resource path.
       Propertynaam moet de naam zijn van een property van de resource of van een property van een gegevensgroep van de resource
 
-  Scenario: url templating is toegepast in de server url
-    Gegeven een OpenAPI specificatie met de volgende servers definitie
-    """
-    servers:
-    - url: https://{domainname}/lvbag/api/haal-centraal-bag-bevragen/v{versie}
-      variables:
-        domainname:
-          enum:
-          - 'test.kadaster.nl'
-          - 'www.kadaster.nl'
-          default: 'www.kadaster.nl'
-        versie:
-          enum:
-          - '1'
-          - '2'
-          default: '1'
-    """
-    Als een consumer de server url wil expanden voor versie 2 op de test omgeving
-    Dan moet hij {domainname} vervangen met test.kadaster.nl
-    En moet hij {versie} vervangen met 2
-    En ziet de ge-expande server url er als volgt uit: https://test.kadaster.nl/lvbag/api/haal-centraal-bag-bevragen/v2
-
-  Scenario: url templating toegepassen voor een externe API url
-    Gegeven de resource path om een ingeschreven persoon te raadplegen is: /ingeschrevenpersonen/{burgerservicenummer}
-    Als de provider URI templating wil toepassen voor de betreffende resource path
-    Dan voegt hij {serverurl} aan het begin van de resource path
-    En ziet de volgende HAL link er als volgt uit
-    | templated | href                                                   |
-    | true      | {serverurl}/ingeschrevenpersonen/{burgerservicenummer} |
-
-  Abstract Scenario: url templating toepassen voor een resource path
-    Gegeven de OpenAPI specificatie met de volgende servers definitie
-    """
-    servers:
-    - url: https://{hostname}/lvbag/api/haal-centraal-bag-bevragen/v{versie}
-      variables:
-        hostname:
-          default: 'www.kadaster.nl'
-        versie:
-          default: '1'
-    """
-    En de path om een adres te raadplegen: '/adressen/{adresidentificatie}'
-    Als de provider URI templating voor een <type> adres url
-    Dan bevat de response de volgende HAL link
-    | templated | href            |
-    | true      | <templated url> |
-    En bevat de response een property met de naam adresidentificatie die moet worden gebruikt om de templated url te expanden
+  Scenario: Verwijzing naar één externe Resource
+    Gegeven een KadastraalOnroerendeZaak heeft een verwijzing naar een Adres met identificatie '0200200000003734'
+    En een Adres is te bevragen bij de BAG API via endpoint '/adressen'
+    Als een templated Hal link voor de Adres is gegenereerd
+    Dan is de Hal link naar de Adres gelijk aan
+    | href                                  | templated |
+    | {serverurl}/adressen/0200200000003734 | true      |
```

Comment:
Hier zou dan toch de placeholder {bagserverurl}  op zijn plaats zijn ?

Reply:
Op één of andere duistere reden lijkt het of je naar een oude commit kijk. Als ik op de 'View changes' button klik, dan zie ik dat je de 3e commit heb gereviewed, maar er zijn 5 commits in de branch